src/Platform_Core/org/lobobrowser/security/PermissionSystem.java

```diff
@@ -203,6 +219,18 @@ public void dump() {
           System.out.print(String.format(" %s%d", permFmtStr, c.parentCells.size()));
         });
       }
+      
+      public String dumpString() {
+    	  StringBuilder rowPermissionState = new StringBuilder();
+          Arrays.stream(requestCells).forEach(c -> {
+            final PermissionResult permissionResult = c.getEffectivePermission();
+            final String permStr = permissionResult.permission.toString().substring(0, 1);
+            final String permFmtStr = permissionResult.isDefault ? " " + permStr + " " : "[" + permStr + "]";
+            rowPermissionState.append(permFmtStr);
+            System.out.print(String.format(" %s%d", permFmtStr, c.parentCells.size()));
```

Comment:
Same here: indentation is wrong in this function.

Also, can you remove the System.out.print() call? That was for debugging right?